.pre-commit-config.yaml

```diff
@@ -0,0 +1,56 @@
+exclude: Pipfile\.lock|migrations|\.idea
+
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.0.1
+    hooks:
+      - id: trailing-whitespace
+      - id: check-executables-have-shebangs
+      - id: check-json
+      - id: check-case-conflict
+      - id: check-toml
+      - id: check-merge-conflict
+      - id: check-xml
+      - id: check-yaml
+      - id: end-of-file-fixer
+      - id: check-symlinks
+      - id: mixed-line-ending
+      - id: fix-encoding-pragma
+        args:
+          - --remove
+      - id: pretty-format-json
+        args:
+          - --autofix
+      - id: requirements-txt-fixer
+
+  - repo: https://github.com/PyCQA/isort
+    rev: 5.9.1
+    hooks:
+      - id: isort
+        name: Run isort to sort imports
+        files: \.py$
+        exclude: ^build/.*$|^.tox/.*$|^venv/.*$
+        args:
+          - --lines-after-imports=2
+          - --multi-line=3
+          - --trailing-comma
+          - --force-grid-wrap=0
+          - --use-parentheses
+          - --ensure-newline-before-comments
+          - --line-length=88
+
+  - repo: https://gitlab.com/pycqa/flake8
+    rev: 3.9.2
+    hooks:
+        - id: flake8
+          args:
+            - --per-file-ignores=*test*:E501,*__init__*:F401,*wsgi.py:E402
+            - --max-line-length=88
+            - --ignore=E203,W503
```

Comment:
It can't 😞 `pre-commit` runs each repo in its own virtualenv and exists ?? somewhere ?? that doesn't include the root files from our repository, so it doesn't have access to our `.flake8` file 😢

Reply:
I see, the multi-project issue, then don't mind my question. Thanks for the clarification.